nimble/host/src/ble_l2cap_sig.c

```diff
@@ -869,6 +1267,164 @@ ble_l2cap_sig_coc_connect(uint16_t conn_handle, uint16_t psm, uint16_t mtu,
     return rc;
 }
 
+int
+ble_l2cap_sig_coc_connect_multiple(uint16_t conn_handle,
```

Comment:
Maybe this should be named ble_l2cap_sig_coc_enhanced_connect()? I think this would be more clear as connect multiple could mean creating multiple legacy channels. And you can still use this API to create only one channel, so... :)